packages/big-design/src/components/Select/styled.tsx

```diff
@@ -34,4 +34,3 @@ export const DropdownButton = styled(StyledButton)`
 StyledStatusMessage.defaultProps = { theme: defaultTheme };
 StyledDropdownIcon.defaultProps = { theme: defaultTheme };
 StyledInputContainer.defaultProps = { theme: defaultTheme };
-DropdownButton.defaultProps = { theme: defaultTheme };
```

Comment:
Why is this getting removed?

Reply:
I keep getting this error on my type 

```
Type of property 'defaultProps' circularly references itself in mapped type 'Pick<ForwardRefExoticComponent<Pick<Pick<any, Exclude<keyof ReactDefaultizedProps<StyledComponentInnerComponent<WithC>, ComponentPropsWithRef<StyledComponentInnerComponent<WithC>>>, StyledComponentInnerAttrs<...> | ... 1 more ... | StyledComponentInnerAttrs<...>> | Exclude<...> | Exclude<...> | Exclude<...>> & Parti...'.
```

Reply:
nvm fixed I was using the wrong component